Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -38,6 +39,12 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
     )
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) = with(binding) {
+
+        startValidationCheck.defaultButton.setOnClickListener {
+            startValidationCheck.isLoading = true
```

Comment:
Does the custom-view persist it's loading state via `SavedState`?
If not and the "loading operation" isn't canceled when the screen is left, then the user could press the home button, come back to the app, and the operation could  still be running, but the loading indicator would be gone.

We'd need to either make sure the view persists it's own state or save it in a instance that survives as long as the "loading operation", e.g. viewmodel?